app/Http/Controllers/LoginController.php

```diff
@@ -38,9 +38,15 @@ public function authenticateWeb(Request $request): RedirectResponse {
         if (Auth::guard('web')->attempt(['staff_username' => $credentials['username'], 'password' => $credentials['password']])) {
             // Authentication passed, redirect:
             $serverId = $request->get("server_id", 0);
+            if($serverId)
```

Comment:
What is the point of this if statement? It needs curly braces around the below puts into the session if it doesn't exist?